packages/core/ui-settings/core-ui-settings-common/src/theme.ts

```diff
@@ -52,6 +52,11 @@ export function parseThemeTags(input?: unknown): ThemeTags {
     return DEFAULT_THEME_TAGS;
   }
 
+  // TODO: remove when Borealis is in public beta
```

Comment:
Can we create an issue for this to not forget to remove this condition? And then update the comment with the link to the issue?

Reply:
Surely! I'm going to create an issue shortly and will update the comment in a follow-up PR so that we don't need to wait for CI to pass again. Thanks for catching it!